build.js

```diff
@@ -45,7 +45,6 @@ Metalsmith(__dirname)
   }))
   .use(msstatic({"src": "styles/", "dest": "ipfs.io/styles"}))
   .use(msstatic({"src": "media/", "dest": "ipfs.io/media"}))
-  .use(msstatic({"src": "blog/", "dest": "ipfs.io/blog"}))
```

Comment:
f**k... not a good idea too... as it would break old links to the blog...

is there a way to redirect them to an other url?